### ISSUE TITLE
plot loss 

### ISSUE THREAD

**Issue by yxyyxy93** [Tournesol-Saturday/DTC_test#2]
train_losses = []
    valid_losses = []

    for epoch in range(num_epochs):
        ....
        train_losses.append(running_loss / len(train_loader))
        
        ...
        model.eval()
        running_val_loss = 0.0
        with torch.no_grad():
            for images, targets in valid_loader:
                images, targets = images.to(device), targets.to(device)
                outputs = model(images)
                loss = criterion(outputs, targets)
                running_val_loss += loss.item()
        valid_losses.append(running_val_loss / len(valid_loader))

    return train_losses, valid_losses

def plot_losses(train_losses, valid_losses):
    plt.figure(figsize=(10, 5))
    plt.plot(train_losses, label='Train Loss')
    plt.plot(valid_losses, label='Validation Loss')
    plt.xlabel('Epoch')
    plt.ylabel('Loss')
    plt.legend()
    plt.title('Loss Curve')
    plt.show()
